app.js

```diff
@@ -37,24 +36,21 @@ app.get("/search/:query", function(req, res) {
         }
     }
 
-    // console.log(process.env.APICREDENTIAL);
     request("https://www.googleapis.com/youtube/v3/search?part=snippet&q=" + finalQuery + "&key=" + process.env.APICREDENTIAL,
     function(error, response, body) {
     	if (error) {
     		console.log(error);
     	}
         var data = JSON.parse(body);
-        // res.send(data);
         console.log(data);
         res.render("index", {data: data});
     }
 );
 });
 
 app.get("/download/:videoUrl", function(req, res) {
-    console.log(cwd);
-    var video = youtube("https://www.youtube.com/embed/" + req.params.videoUrl, 
-    ["--format=18"], 
+    const video = youtube("http://www.youtube.com/watch?v=" + req.params.videoUrl, 
```

Comment:
Let this be var, for now, I suspect problems in asynchronous nature if the variable is constant.